docs/devguide/run_time_configuration.rst

```diff
@@ -3483,7 +3485,7 @@ It can also be set in application code using:
 
 The *level* defaults to a value of 0 and has values of 0 to 10 as defined below:
 
-* 0 – debug logging is disabled
+* 0 -- debug logging is disabled
```

Comment:
```suggestion
* 0 - debug logging is disabled
```
inconsistent with the items below (so perhaps change those instead?)

Reply:
Many of these were just search and replace, but I didn't consider they'd be inconsistent, which I should've known.